staging/src/k8s.io/kubectl/pkg/cmd/drain/drain_test.go

```diff
@@ -867,7 +867,7 @@ func TestDrain(t *testing.T) {
 				switch {
 				case recovered != nil && !sawFatal:
 					t.Fatalf("got panic: %v", recovered)
-				case test.expectFatal && !sawFatal:
+				case test.expectDelete && test.expectFatal && !sawFatal:
```

Comment:
Are there any extra test cases? I think we should add a test case which would fail without the bug fix, but works because  of the bug fix.

Reply:
Helper#GetPodsForDeletion is called by:
```
func RunNodeDrain(drainer *Helper, nodeName string) error {
```
which is not called in any test.
How about adding new test in a follow-on issue ?